org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/Painter.java

```diff
@@ -22,6 +23,7 @@ public Painter() {
     this.remainingPaint = 0;
     World worldInstance = World.getInstance();
     this.grid = worldInstance.getGrid();
+    this.hasInfinitePaint = (this.grid.getSize() >= LARGE_GRID_SIZE) ? true : false;
```

Comment:
nit: can simplify to `this.hasInfinitePaint = this.grid.getSize() >= LARGE_GRID_SIZE;`